rust/update-conan-recipe-and-create-pull-request/src/decode_github_content.rs

```diff
@@ -1,17 +1,29 @@
-use base64::Engine;
-use crate::add_version_to_conandata_yml::AddVersionToConandataYmlArgument;
+use {crate::Result, base64::DecodeError};
 
+#[deprecated]
 pub struct DecodeGithubContentArgument<'a> {
     pub content: &'a String,
```

Comment:
To pass string as view you have to decide: 
- you should pass as `&str` if you want only read some data
- pass as `String` if you want to use allocated space in this string